packages/mirinae/src/data-display/dynamic/dynamic-layout/templates/raw/index.vue

```diff
@@ -60,8 +60,8 @@ export default {
             layoutName: computed(() => (props.options.translation_id ? vm.$t(props.options.translation_id) : props.name)),
             rootData: computed<any[]>(() => {
                 if (props.options.root_path) {
-                    const rootData = getValueByPath(props.data, props.options.root_path) ?? [];
-                    return Array.isArray(rootData) ? rootData : [rootData];
+                    const rootData = getValueByPath(props.data, props.options.root_path) ?? {};
+                    return rootData;
                 }
```

Comment:
nitpick: It seems like the type of `rootData` should be changed from `any[]` to `any`.

Reply:
Oh, I missed checking the type. Thanks!